README.md

```diff
@@ -21,11 +21,11 @@ to install the npm version, run the following
 npm install npm@8.6.0 -g
 
 ```bash
-npm install --legacy-peer-deps
+npm install
```

Comment:
since we are not using vueuse any more, we no longer have the  conflict that came with it